src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -710,18 +710,23 @@ public void connect(final AWSIotMqttClientStatusCallback statusCallback) {
         // create a thread as the credentials provider getCredentials() call may require
         // a network call and will possibly block this connect() call
         new Thread(new Runnable() {
-            @Override
-            public void run() {
-                if (mqttLWT != null) {
-                    options.setWill(mqttLWT.getTopic(), mqttLWT.getMessage().getBytes(),
-                            mqttLWT.getQos().asInt(), false);
-                }
-                AWSIotWebSocketUrlSigner signer = new AWSIotWebSocketUrlSigner("iotdata");
-                String mqttWebSocketURL = signer.getSignedUrl(webSocketEndpoint,
-                        clientCredentialsProvider.getCredentials(), System.currentTimeMillis());
-                options.setServerURIs(new String[] {mqttWebSocketURL});
-                mqttConnect(options, statusCallback);
-            }
+			@Override
```

Comment:
Please don't make formatting changes.
If you do, then undo them so that diff is cleaner

Reply:
Right. I normally avoid these.  I had to add alll to try/catch so I did an auto reformat in eclipse to indent correctly. Looks like I selected 2 lines above and probably one line below.  I will try to update this after I get your feedback on the Exception issue above. Thanks!